lib/rspec/core/configuration.rb

```diff
@@ -778,24 +780,58 @@ def full_backtrace=(true_or_false)
         @backtrace_formatter.full_backtrace = true_or_false
       end
 
-      # Returns the configuration option for color, but should not
-      # be used to check if color is supported.
+      # Enables color output if the output is a TTY.  As of RSpec 3.6, this is
+      # the default behavior and this option is retained only for backwards
+      # compatibility.
       #
+      # @deprecated No longer recommended because of complex behavior. Instead,
+      #   rely on the fact that TTYs will display color by default, or set
+      #   {#color_mode} to :forced to display color on a non-TTY output.
+      # @see color_mode
       # @see color_enabled?
       # @return [Boolean]
       def color
         value_for(:color) { @color }
       end
 
+      # Sets the color mode to one of:
```

Comment:
I'm all for improving the docs.  I'd just warn that YARD doesn't always render things how you think it will, and ultimately what we care the most about is how it looks when rendered -- so please run a local yard server (using `bundle exec yard server --reload`) while editing so you can take a look at the changes and ensure they render well.